docs/sphinx/install.rst

```diff
@@ -586,6 +586,93 @@ inside the container. Note that most Python packages that use GPU-acceleration,
 
     python3 -m pip install cupy-cuda11x
 
+.. _distributed-computing-with-mpi:
+
+Distributed Computing with MPI
+------------------------------------
+
+CUDA Quantum supports the Message Passing Interface (MPI) parallelism via a runtime configurable plugin interface.
+Specifically, while a builtin plugin is shipped with the CUDA Quantum Docker image, users can also activate 
+a custom CUDA Quantum MPI plugin targeting their local MPI installation without recompiling or reinstalling the stack.
+
+CUDA Quantum MPI API for C++ and Python will then be delegated 
+to the plugin implementation selected at runtime.  
+
+The following table summarizes mechanisms whereby a CUDA Quantum MPI plugin can be picked up.
+
+.. tab:: Builtin
+
+  **Use case**: build from source, Docker containers
+
+  When a local MPI installation is detected at build time, CUDA Quantum will build an MPI plugin targeting this implementation when build from source.
+  Similarly, the :ref:`CUDA Quantum Docker image <install-docker-image>` is shipped with a builtin MPI plugin based on its optimized OpenMPI installation inside the image. 
+  No action is required to use this builtin plugin. 
+
+  .. note::
+
+    Manually-activated MPI plugin takes precedence over builtin plugin. 
+    For instance, the user may choose to reinstall a different MPI library inside the Docker container and 
+    activate a new CUDA Quantum MPI plugin. The newly-activated plugin will always be used even though a builtin one 
+    was shipped with the Docker image. 
+
+.. tab:: Manual Activation
+
+  **Use case**: data centers with custom MPI installation, CUDA Quantum binary distributions without MPI support (including Python wheels).
+
+  **Instructions:**
+
+  (1) Locate the CUDA Quantum `distributed_interfaces` sub-directory at the top-level of the CUDA Quantum installation directory.
+
+  (2) Execute the `activate_custom_mpi.sh` bash script.
```

Comment:
What is there are more than one local MPI implementation available? (is that possible?)
Is there a way to specify which one we want to use?